### PR TITLE
[ADD] DCOS-1323 

### PR DESCRIPTION
**Pull request by truskovskiyk**
dcos help <subcommand> should be aliased to dcos <subcommand> --help

### CODE REVIEW

**Comment by ssk2** [comment]
Looks good, thanks :)

File: cli/dcoscli/help/main.py [hunk 1]
```diff
@@ -45,33 +46,69 @@ def _cmds():
...
+
+def _help_command(command):
+    """
+    :returns: process return code
+    :rtype: int
+    """
+
+    try:
+        module = "dcoscli.{}.main".format(command)
+        emitter.publish(__import__(module, fromlist=['']).__doc__)
```
Comment (jsancio) [inline cli/dcoscli/help/main.py]: Let's avoid importing __xxx and _xx symbols. The developer is going to assume that no one outside the module is using them.

Comment (jsancio) [inline cli/dcoscli/help/main.py]: Is this mechanism going to work for commands that were installed? For example the user is able to install the `kafka` subcommand with `dcos package install --cli kafka`. If the user does that will `dcos help kafka` work?

File: cli/tests/integrations/test_help.py [hunk 2]
```diff
@@ -7,6 +7,7 @@ def test_help():
 Usage:
     dcos help --info
     dcos help
+    dcos help <command>
```
Comment (jsancio) [inline cli/tests/integrations/test_help.py]: We need more tests. We need to test success cases and failure cases. For example: `dcos help marathon`, `dcos package install kafka --cli && dcos help kafka`, `dcos help unknown-command`.

File: cli/dcoscli/help/main.py [hunk 3]
```diff
@@ -45,33 +46,69 @@ def _cmds():
...
+        emitter.publish(commands_message)
+        emitter.publish(
+            "\nGet detailed command description with"
+            " 'dcos <command> --help'.")
 
+        return 0
+
+
+def _help_command(command):
+    """
```
Comment (jsancio) [inline cli/dcoscli/help/main.py]: We are missing pydoc for the arguments: `command`.

File: cli/dcoscli/help/main.py [hunk 4]
```diff
@@ -45,33 +46,69 @@ def _cmds():
...
 
-    directory = util.dcos_path()
-    logger.debug("DCOS Path: {!r}".format(directory))
 
-    paths = subcommand.list_paths()
-    with ThreadPoolExecutor(max_workers=len(paths)) as executor:
-        results = executor.map(subcommand.documentation, paths)
-        commands_message = options.make_command_summary_string(sorted(results))
+def _help(command):
+    """
```
Comment (jsancio) [inline cli/dcoscli/help/main.py]: We are missing pydoc for the arguments: `command`.

**Comment by jsancio** [comment]
Thanks for the PR. Added some comments.

**Pull request merged**